### DISCUSSION TITLE
How to construct `iced::window::Icon` in 0.9

### DISCUSSION THREAD

**Discussion by fenhl** [hecrj/iced#1806] [Q&A] [answered]
In iced 0.8 I was able to use the `iced::window::Icon::from_rgba` function to supply the `icon` field of `iced::window::Settings`. This function no longer exists in iced 0.9. There is no mention of this in [the changelog](https://github.com/iced-rs/iced/blob/master/CHANGELOG.md#090---2023-04-13).

**Comment by carl-anders** [comment, marked as answer]
Use `iced::window::icon::from_rgba` instead. So instead of `Icon` it's `icon`.
